docs/index.md

```diff
@@ -1,13 +1,94 @@
-El Collectooorr aims to autonomously create collections of NFTs by minting them as they drop on Art Blocks. It uses complex logic to decide which mints to participate in and what prices to bid at, so users don’t have to. The service is fueled by the community of users who fund collections. Each collection is designed to have its own unique ERC-20 token, for users to vote and collectively decide on what happens to the collection.
+![MintKit](images/el-collectooorr.png){ align=left width="150" }
+The MintKit is a toolkit to build services with minting capabilities. For example, El Collectooorr service, which is based on the MintKit, aims to autonomously create collections of NFTs by minting them as they drop on Art Blocks. It uses complex logic to decide which mints to participate in and what prices to bid at, so users don’t have to. The service is fueled by the community of users who fund collections. Each collection is designed to have its own unique ERC-20 token, for users to vote and collectively decide on what happens to the collection.
 
+## Demo
 
-##Run the code
+!!! warning "Important"
 
-!!! info
-	This section will be added soon.
+	This section is under active development - please report issues in the [Autonolas Discord](https://discord.com/invite/z2PT65jKqQ).
 
+Once you have {{set_up_system}} to work with the Open Autonomy framework, you can run a local demo of the El Collectooorr service as follows:
 
-##Build your own
+1. Fetch the El Collectooorr service.
 
-!!! info
-	This section will be added soon.
+	```bash
+	autonomy fetch valory/elcollectooorr:0.1.0:bafybeidzyd2yi3igjinpi7bwnyal235bmvovuyxy4qrxnitah6fqysemxq --service
```

Comment:
This is not congruent with what's currently in `packages.json`

Reply:
Fixed.